src/game/option/option_control_pick.c

```diff
@@ -0,0 +1,116 @@
+#include "config.h"
+#include "game/gameflow.h"
+#include "game/input.h"
+#include "game/music.h"
+#include "game/option/option_sound.h"
+#include "game/sound.h"
+#include "game/text.h"
+#include "global/vars.h"
+
+#include <stdio.h>
+
+#define MIN_VOLUME 0
+#define MAX_VOLUME 10
+
+typedef enum PICK_TEXT {
+    TEXT_KEYBOARD = 0,
+    TEXT_CONTROLLER = 1,
+    TEXT_TITLE = 2,
+    TEXT_TITLE_BORDER = 3,
+    TEXT_NUMBER_OF = 4,
+    TEXT_OPTION_MIN = TEXT_KEYBOARD,
+    TEXT_OPTION_MAX = TEXT_CONTROLLER,
+} PICK_TEXT;
+
+static TEXTSTRING *m_Text[TEXT_NUMBER_OF] = { 0 };
+
+static void Option_ControlPickInitText(void);
+static void Option_ControlPickShutdownText(void);
+
+static void Option_ControlPickInitText(void)
+{
+    char buf[20];
+
+    m_Text[TEXT_TITLE_BORDER] = Text_Create(0, -32, " ");
+    Text_AddBackground(m_Text[TEXT_TITLE_BORDER], 180, 85, 0, 0, TS_BACKGROUND);
+    Text_AddOutline(m_Text[TEXT_TITLE_BORDER], true, TS_BACKGROUND);
+
+    m_Text[TEXT_TITLE] =
+        Text_Create(0, -30, g_GameFlow.strings[GS_CONTROL_CUSTOMIZE]);
+    Text_AddBackground(m_Text[TEXT_TITLE], 176, 0, 0, 0, TS_HEADING);
+    Text_AddOutline(m_Text[TEXT_TITLE], true, TS_HEADING);
+
+    m_Text[TEXT_KEYBOARD] =
+        Text_Create(0, 0, g_GameFlow.strings[GS_CONTROL_KEYBOARD]);
+    m_Text[TEXT_CONTROLLER] =
+        Text_Create(0, 25, g_GameFlow.strings[GS_CONTROL_CONTROLLER]);
+
+    Text_AddBackground(m_Text[g_OptionSelected], 128, 0, 0, 0, TS_REQUESTED);
+    Text_AddOutline(m_Text[g_OptionSelected], true, TS_REQUESTED);
+
+    for (int i = 0; i < TEXT_NUMBER_OF; i++) {
+        Text_CentreH(m_Text[i], 1);
+        Text_CentreV(m_Text[i], 1);
+    }
+}
+
+static void Option_ControlPickShutdownText(void)
+{
+    for (int i = 0; i < TEXT_NUMBER_OF; i++) {
+        Text_Remove(m_Text[i]);
+        m_Text[i] = NULL;
+    }
+}
+
+CONTROL_MODE Option_ControlPick(INVENTORY_ITEM *inv_item)
```

Comment:
`inv_item` is not used and as such this can take no arguments (remember to use `void`, not empty braces)

Reply:
Ok. I kept it in there since the other inventory functions all take it even though only the option_passport uses it.